src/exception/treatment-detail-exceptions/service.ts

```diff
@@ -75,6 +76,15 @@ export const create = async ({
 	}
 };
 
+export const getTreatmentDetailException = async ({
+	programId,
+}: {
+	programId: string;
+}): AsyncResult<TreatmentDetailException> => {
+	const result = await getByProgramId(programId);
+	return result ? result : failure(`Cannot find program exceptions for ${programId}`);
```

Comment:
I think you're always going to get a `result` so your `failure` return will never happen
the result will either be a `Success` or a `Failure` type

Likely you want to add a check for `success` prop of `result`
ref: https://github.com/icgc-argo/argo-clinical/blob/c6fc571f100d12f1dc3f9bde3e67a9a0e670da82/src/utils/results.ts#L9

Reply:
Right, thanks, updated
I check for `success` elsewhere and must have overlooked this early change

https://github.com/icgc-argo/argo-clinical/pull/1171/files#diff-1d560ee936a20e4760949fc5abd3a45833ad9894bac7f4b1909684fcaca78b5eR119